### PR TITLE
Update deployment configuration

### PR DESCRIPTION
**Pull request by wowkalucky**
- documentation updates added;
- pinned python requirements versions;
- updated production bridge service Dockerfile;
- updated nginx config (static files serving);
- updated Django settings - staticfiles, security,  (see: https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/#deployment-checklist)
- content-source views moved under login_required;
- changed python app wsgi port from 8000 to 8888 (for local development convenience);
- template fix for demo-issue;

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ Local deployment contains two containers:
 
   Volume "pgs" is added to the the database container.
 
+  Note: Development server available on `localhost:8888`
```
Comment (kunanit) [inline README.md]: See other comment on port number

File: bridge-adaptivity/docker-compose_local.yml [hunk 2]
```diff
@@ -14,11 +14,11 @@ services:
     build:
       context: .
       dockerfile: Dockerfile_local
-    command: bash -c "sleep 5 && python manage.py runserver 0.0.0.0:8000"
+    command: bash -c "sleep 5 && python manage.py runserver 0.0.0.0:8888"
     volumes:
       - .:/bridge-adaptivity
     ports:
-      - "8000:8000"
+      - "8888:8888"
```
Comment (kunanit) [inline bridge-adaptivity/docker-compose_local.yml]: Sorry - Can we change the port, or at least the host port being mapped to, to something besides 8888? (8888 is the default for jupyter notebooks)

Comment (wowkalucky) [inline bridge-adaptivity/docker-compose_local.yml]: oh..I'll change it back! There is no a big problem to configure a dev port...

**Comment by wowkalucky** [comment]
Changed port number back to 8000 - please, have a look.

**Pull request merged**